packages/components/src/input-control/types.ts

```diff
@@ -105,7 +122,18 @@ export interface InputFieldProps extends BaseProps {
 	paddingInlineStart?: CSSProperties[ 'paddingInlineStart' ];
 	paddingInlineEnd?: CSSProperties[ 'paddingInlineEnd' ];
 	setIsFocused: ( isFocused: boolean ) => void;
+	/**
+	 * Whether to wrap input field in a Tooltip component.
```

Comment:
A bit of a nit, but maybe it's better not to expose implementation details to our documentation (in case we ever change how this feature is achieved)

```suggestion
	 * Whether to wrap input field in a tooltip.
```

Reply:
👍 Good call. Thanks. https://github.com/WordPress/gutenberg/pull/42726/commits/8c0acf553ad986880561af11323deabf73e70c17